### ISSUE TITLE
Question: Windows auth flow, can be more simple?

### ISSUE THREAD

**Issue by iggy32662** [IdentityServer/IdentityServer4#1698]
Regarding strictly Windows auth, the quickstart UI project Account controller has an external login action that initiates the authentication challenge on HttpContext, captures the result cookie, signs the user in, captures that cookie, then deletes the first temporary cookie. Is it too far fetched to have an MVC app, with Windows auth enabled, simply call SignInAsync with the windows principal and skip the other steps? 

The bottom line is I want a landing page internal to my domain that users can go to, that will get an id token and sign-in cookie, then get immediately redirected to the real MVC application that is both internally and externally accessible. The intent is to then start the hybrid flow from there to invoke APIs.

**Comment by leastprivilege** [comment]
You can simpify by calling ChallengeAsync directly when loading the login page - and then AuthenticateAsync to get the windows identity.

**Comment by leastprivilege** [comment]
Any update on the issue? closing for now - feel free to re-open if it needs further discussion.
